### ISSUE TITLE
Show interpreter errors

### ISSUE THREAD

**Issue by wvengen** [wvengen/netswarm-simulator#1]
Compiler errors are shown, but interpreter errors (like an unknown function) not. That would be useful.

**Comment by SebastianZug** [comment]
That's right. We integrated your implementation in our eLearning system and looking forward to it :-)

Description file of the course
https://github.com/liaTemplates/netswarm-simulator

Resulting interactive environment 
https://liascript.github.io/course/?https://raw.githubusercontent.com/liaTemplates/netswarm-simulator/master/README.md#3
